lib/iris/__init__.py

```diff
@@ -221,15 +225,8 @@ def context(self, **kwargs):
         statement, the previous state is restored.
 
         For example::
-
-            with iris.FUTURE.context():
-                iris.FUTURE.cell_datetime_objects = True
-                # ... code which expects time objects
-
-        Or more concisely::
-
-            with iris.FUTURE.context(cell_datetime_objects=True):
-                # ... code which expects time objects
+            with iris.FUTURE.context(cell_datetime_objects=False):
```

Comment:
Wasn't a necessary change, but a welcome one.

Reply:
My thinking was that the docs would otherwise technically not have been accurate...